spec/Section 3 -- Type System.md

```diff
@@ -837,7 +837,6 @@ input ExampleInputObject {
 Original Value          | Variables       | Coerced Value
 ----------------------- | --------------- | -----------------------------------
 `{ a: "abc", b: 123 }`  | {null}          | `{ a: "abc", b: 123 }`
-`{ a: 123, b: "123" }`  | {null}          | `{ a: "123", b: 123 }`
```

Comment:
I think that distinction is a little too subtle. Whether coercion fails because you didn't provide a required value, or because you provided one of the wrong type, the user is going to see an error, and probably will care more about the fact that it didn't work than whether it is technically a "coercion" error. Also, note that even though "coercion" is narrowly defined in the spec, it also has a more general meaning or translation from one type to another which many readers will have in mind when they look at this table (and might also explain why this example was here in the first place, even if it was wrong).